src/Data/Codec/Argonaut/Common.purs

```diff
@@ -80,5 +80,5 @@ map codecA = dimap M.toUnfoldable M.fromFoldable <<< array <<< tuple codecA
 -- | A codec for `StrMap` values.
 -- |
 -- | Encodes as an array of two-element key/value arrays in JSON.
-strMap ∷ ∀ a. JsonCodec a → JsonCodec (SM.StrMap a)
-strMap = dimap SM.toUnfoldable SM.fromFoldable <<< array <<< tuple string
+strMap ∷ ∀ a. JsonCodec a → JsonCodec (FO.Object a)
+strMap = dimap FO.toUnfoldable FO.fromFoldable <<< array <<< tuple string
```

Comment:
Maybe we should rename this to `object` too?

Reply:
Good catch! I chose foreignObject, since object is already taken